docs/organizations/overview.mdx

```diff
@@ -5,13 +5,20 @@ description: Learn about how to create and manage Clerk organizations and their
 
 # Organizations
 
-Organizations are a way of grouping your application's users. Organizations are shared accounts, useful for project and team leaders—think of GitHub Teams or the different departments of a company. Members of different organizations can usually collaborate across shared resources but might have different levels of access and permissions with different [role](/docs/organizations/overview#roles).
+Organizations are a way of grouping your application's users. Organizations are shared accounts, useful for project and team leaders—think of GitHub Teams or the different departments of a company. Members of different organizations can usually collaborate across shared resources but might have different levels of access and permissions with different [roles](/docs/organizations/overview#roles).
 
-To enable organizations, go to the Clerk Dashboard and navigate to [**Organizations Settings**](https://dashboard.clerk.com/last-active?path=organizations-settings). Select the toggle next to **Enable Organizations**.
+There is no limit to the number of organizations a user can be a member of. However, a user can only create up to 100 organizations in a given application instance.
 
-There is no limit to the number of organizations a user can be a member of. However, a user can only create up to 100 organizations in a given application instance. All members of an organization can view information about other members in the same organization.
+## Enable organizations in your application
 
-Clerk provides [prebuilt components](/docs/components/overview), [React hooks](/docs/references/react/use-organization), and [APIs](/docs/references/javascript/organization/organization) to help you integrate organizations into your application. Using the [Organizations Backend API](https://clerk.com/docs/reference/backend-api/tag/Organizations), you can provide additional [private and public metadata for the organization](/docs/organizations/metadata) and set custom attributes. 
+To enable organizations:
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings).
+2. Select your application, then select **Organizations Settings** from the navigation sidebar.
```

Comment:
```suggestion
2. Select your application, then, in the navigation sidebar, select **Organizations Settings**.
```

Based on the style guide rule to put the destination of the action first

Reply:
ahh then we need to update it everywhere else as this was your addition to the nextjs quickstart and I followed suit from that. I can make a PR to update all these instances.

Reply:
Okay, thanks 👍